KK 004 -  Home Page - Changed logo and welcome msg🐨

eleniaslanidou89 opened the pull request:
Home Page - Changed logo and welcome msg🐨
![image](https://user-images.githubusercontent.com/57090719/140620151-81639924-26a1-4813-a2e4-54bf78a38cef.png)

eleniaslanidou89 committed:
📣 added component Description in Aboout page

eleniaslanidou89 committed:
Revert "📣 added component Description in Aboout page"

This reverts commit 088b98c62b2c52e5febb48c0403819cb3f3c0be3.

eleniaslanidou89 committed:
replaced logo with a new logo-img 🐨

eleniaslanidou89 committed:
new logo

eleniaslanidou89 committed:
changed logo

kgkoutio commented on src/components/HomePage.vue:
```diff
@@ -50,7 +50,8 @@ export default {
 <style scoped>
 .logo {
-  border-radius: 50%;
+  width: 250px;
```
Is it a common practice in web development to set images with fixed width and fixed height? How does that scale in the mobile browsers where there might not be enough space?

eleniaslanidou89 replied:
I agree with you @kgkoutio branch name should contain a descriptive message. 
According to the width and height in web development it is usually used to set a width and a height and then change it in the responsive part. I haven't add the responsive part yet. In the responsive part we can set the size of the image for a Tablet version ,mobile version etc. This is based my knowledge in React framework, I will check if there are differences in Vuejs.
Example: 
/***************   MOBILE VIEW   ***************/
.nav_logo {
    margin: 0 25px 0 10%;
    height: 60px;
  }

kgkoutio reviewed (commented):
Hey @eleniaslanidou89 ! Great work with changing the logo 💯  I have a few thoughts when it comes to the code review based on my experience:
Branch naming is good to contain the ticket id but also a descriptive message, as it will be also humanly readable. This will help the reviewer to check out the right branch, and avoid mistakes. In that case I would name the branch `KK-004-new-logo`. What do you think? (Of course you don't have to change it now :) )

The other thought I have is added as a question.

kgkoutio commented on src/assets/giga-vue.svg:
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 364 476"><defs><style>.cls-1{fill:#fff;}.cls-2{fill:#34495e;}.cls-3{fill:#41b883;}</style></defs><g id="Page-1"><g id="tamagotchi"><path id="Shape" d="M206,51.57a31,31,0,1,0-48,0C67.73,70.29.68,191.45.68,296.93.68,405.1,71.84,475,182,475s181.31-69.9,181.31-178.07C363.31,191.45,296.26,70.29,206,51.57ZM165,32a17,1 ...
```
Ah I forgot to ask about this one. Excuse my ignorance, but what is this for?

eleniaslanidou89 replied:
The logo that we have is a combination of an img and a logo that I created. This svg is the img without the logo. I will delete it, we don't need it.

kkarayannis commented:
When there are visual changes, a screenshot in the PR helps the reviewer a lot 🙂

eleniaslanidou89 committed:
deleted img

eleniaslanidou89 committed:
changed the name of classes and added margin

eleniaslanidou89 committed:
Changed the message

eleniaslanidou89 committed:
Changed color--> green

eleniaslanidou89 committed:
📣  changed the names of the classes

eleniaslanidou89 committed:
improved structure

eleniaslanidou89 commented:
![image](https://user-images.githubusercontent.com/57090719/141831492-3566b86a-1dd0-4329-bbab-9091af33ee4f.png)

Changes on Home Page:
- Deleted the useless logo.
- Changed the welcome text and the styling of it.
- Changed the names of the classes (based on BEM).
- Improved the structure.
